Tests/Unit/Module/STIG.DomainName.tests.ps1

```diff
@@ -0,0 +1,92 @@
+#region Header
+. $PSScriptRoot\.tests.header.ps1
+#endregion
+
+$domainName = 'Domain.test'
+$forestName = 'Forest.test'
+$domainDistinguished = "DC={0},DC={1}" -f $domainName.Split(".")
+$forestDistinguished = "DC={0},DC={1}" -f $forestName.Split(".")
+$domainDistinguishedPart1 = "DC=Domain"
+$domainDistinguishedPart2 = "DC=Domain,DC=Test"
+$parts = @("Domain","Test")
+
+Describe 'Get-DomainName' {
+        # Test parameter validity -OutputPath
+        It 'Should return Domain FQDN' {
+            Get-DomainName -DomainName $DomainName -Format 'FQDN' | Should -Match $domainName
+        }
+
+        It 'Should return Forest FQDN' {
+            Get-DomainName -ForestName $ForestName  -Format 'FQDN' | Should -Match $forestName
```

Comment:
There's an extra space between $ForestName and -Format.  Line 28 as well.
Also, the variable names in $ForestName and $DomainName should be pascal case, i.e. $forestName, $domainName.

Reply:
Updated